maths/volume.py

```diff
@@ -216,6 +233,7 @@ def main():
     print("Conical Frustum: " + str(vol_conical_frustum(2, 2, 4)))  # ~= 58.6
     print("Spherical cap: " + str(vol_spherical_cap(1, 2)))  # ~= 5.24
     print("Spheres intersetion: " + str(vol_spheres_intersect(2, 2, 1)))  # ~= 21.21
+    print("Tetrahedron: " + str(vol_tetrahedron(2))) # ~=0.94
```

Comment:
Err, forgot a small thing: put two spaces between code and comment, something like:
`print("Tetrahedron: " + str(vol_tetrahedron(2)))  # ~=0.94`
Currently, you have only 1 space which pylint does not like :)